browserApp/model/file.py

```diff
@@ -46,8 +46,13 @@ def get_info(self):
         if os.path.isfile(self.full_path):
             stat = os.stat(self.full_path)
             data['type'] = "File"
-            data['file_size'] = str(stat.st_size)
-            data['created'] = str(stat.st_ctime)
+            # data['file_ext'] = # Retrieve file ext
+            data['file_size'] = str(stat.st_size) + " bytes"
+            data['created'] = datetime.fromtimestamp(stat.st_ctime).strftime('%Y-%b-%d %H:%M')
+            data['modified'] = datetime.fromtimestamp(stat.st_mtime).strftime('%Y-%b-%d %H:%M')
```

Comment:
perhaps you could have a format_time function to make this more readable?
def format_time(time_value):
  return datetime.fromtimestamp(time_value).strftime('%Y-%b-%d %H:%M')
data['created'] = format_time(stat.st_ctime)

Reply:
That's a good idea - Thanks!